notion/block.py

```diff
@@ -772,6 +777,12 @@ class CollectionViewPageBlock(CollectionViewBlock):
         api_to_python=add_signed_prefix_as_needed,
         python_to_api=remove_signed_prefix_as_needed,
     )
+    
+     cover = field_map(
```

Comment:
Very subtle but I think this line's indentation needs fixing

Reply:
Thank you!